hydradx/model/amm/omnipool_amm.py

```diff
@@ -1106,6 +1106,9 @@ def remove_liquidity(self, agent: Agent, quantity: float, tkn_remove: str):
             agent.holdings['LRNA'] = 0
         agent.holdings['LRNA'] += delta_qa
         agent.holdings[(self.unique_id, tkn_remove)] -= quantity
+        if agent.holdings[(self.unique_id, tkn_remove)] == 0:
```

Comment:
So the reason I haven't done this before is because it can be annoying when you want to graph things later. If I were to do, for example, plt.plot([event.agents['agent'].holdings[(self.unique_id, tkn_remove)] for event in events]), that would fail with a KeyError. Everywhere else in the code, we've treated holdings[tkn] == 0 and tkn not in holdings as equivalent. Is it necessary to do this now, or is there another way?